src/openrct2/interface/Screenshot.cpp

```diff
@@ -290,7 +291,7 @@ static int32_t GetHighestBaseClearanceZ(const CoordsXY& location)
             z = std::max<int32_t>(z, element->GetClearanceZ());
         } while (!(element++)->IsLastForTile());
     }
-    return z;
+    return z + 16;
```

Comment:
This was a mistake from my side. I changed that value at some point in an attempt to fix a cropping issue at the top of the image. This is no longer required since the cropping issue was fixed differently, this was reverted to the original state.

Reply:
Turns out some sprites are cropped of with the original z value. This padding has been added to the new function GetTallestVisibleTileTopLimitedm, which is added after the function call of GetHighestBaseClearanceZ
`            int32_t z = GetHighestBaseClearanceZ(location) + 16;`